src/components/Person/Person.jsx

```diff
@@ -1 +1,28 @@
 // export const Person = ({ person }) => ();
+export const Person = ({ person }) => {
+  const {
+    name,
+    age,
+    sex,
+    isMarried,
+    partnerName,
+  } = person;
+
+  const partner = sex === 'm' ? 'wife' : 'husband';
+  const spouse = isMarried ? `${partnerName} is my ${partner}` : 'I am not married';
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">
+        {`My name is ${name}`}
+      </h2>
+      { age && (
```

Comment:
if `age === 0` then the `0` will be displayed before the `p` tag.
```suggestion
      { age  !== undefined && (
```